pkg/controller/volume/persistentvolume/framework_test.go

```diff
@@ -595,22 +599,62 @@ func newTestController(kubeClient clientset.Interface, volumeSource, claimSource
 	if classSource == nil {
 		classSource = fcache.NewFakeControllerSource()
 	}
-
+	// Pass in shared informers as usual but do not start them. Instead,
```

Comment:
Not too hard I don't think but I don't think moving should be in the scope of or block this pr. I was focused on making the tests work again without changing anything, aware that the current model is maybe obsolete (e.g. https://github.com/kubernetes/kubernetes/blame/master/pkg/controller/volume/persistentvolume/testing/fake_controller_source.go is 2+ yrs old)

@jsafrane wrote pretty much all the unit tests and the framework...do you have an opinion on the matter? :) I am happy to attempt to revamp the unit tests if you'll allow it.